pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchConfig.java

```diff
@@ -52,6 +52,15 @@
     )
     private String indexName;
 
+    @FieldDoc(
+            required = false,
```

Comment:
Please align the indentation level with other annotations

Reply:
Fixed it, thanks!